core/text/text_and_dictation.py

```diff
@@ -64,6 +77,56 @@ def prose_number(m) -> str:
     return str(m)
 
 
+@mod.capture(
+    rule="<user.number_string> [(dot | point) <digit_string>] percent [sign|sine]"
+)
+def prose_percent(m) -> str:
+    s = m.number_string
+    if hasattr(m, "digit_string"):
+        s += "." + m.digit_string
+    return s + "%"
+
+
+@mod.capture(
+    rule="<user.number_string> {user.currency_denomination} [[and] <user.number_string> [cents|pence]]"
+)
+def prose_money(m) -> str:
```

Comment:
Should we be consistent in using money or currency?

Reply:
I settled on currency, fixed